packages/runtime-core/src/vnode.ts

```diff
@@ -691,7 +691,7 @@ export function createCommentVNode(
   asBlock: boolean = false
 ): VNode {
   return asBlock
-    ? (openBlock(), createBlock(Comment, null, text))
+    ? (openBlock(),createBlock(Comment, null, text))
```

Comment:
make sure to run prettier on this kind of changes

Reply:
ok, thanks for your reply should i close this pr ?

Reply:
it would be better to fix them and push the changes